judge/forms.py

```diff
@@ -6,16 +6,19 @@
 from django import forms
 from django.conf import settings
 from django.contrib.auth.forms import AuthenticationForm
+from django.contrib.auth.validators import UnicodeUsernameValidator
 from django.core.exceptions import ValidationError
+from django.core.mail import EmailMessage
 from django.core.validators import RegexValidator
 from django.db.models import Q
 from django.forms import BooleanField, CharField, ChoiceField, Form, ModelForm, MultipleChoiceField
-from django.urls import reverse_lazy
+from django.urls import reverse, reverse_lazy
```

Comment:
These imports should be placed right before the class if we are to apply this as a patch in prod. Otherwise changing imports will immediately break the patch and cause issues in prod.